coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -21,7 +21,7 @@ class DocstyleDefinition:
 
     @enforce_signature
     def __init__(self, language: str, docstyle: str, markers: (Iterable, str),
-                 metadata: Metadata):
+                 metadata: Metadata, padding: (Iterable, str)):
```

Comment:
yes this breaks the API. I tried to provide a default value. But the thing is `load` returns an empty tuple if it finds nothing which overwrites that default value. The best solution to providing default values through https://github.com/coala/coala/pull/4557/files#diff-bb503d77ea6d0498c9865cab08c904f4R64